### PR TITLE
docs: warn users that IPsec and KPR are mutual exclusive

### PR DESCRIPTION
**Pull request by f1ko**
Warning users that IPsec and KPR are mutual exclusive.

### CODE REVIEW

File: Documentation/security/network/encryption-ipsec.rst [hunk 1]
```diff
@@ -10,6 +10,13 @@
 IPsec Transparent Encryption
 ****************************
 
+.. warning::
+
+   IPsec does not work when using :ref:`kube-proxy replacement <kubeproxy-free>`.
+   Be aware that other features may require a kube-proxy free environment in
+   which case they are mutual exclusive.
```
Comment (pchaigno) [inline Documentation/security/network/encryption-ipsec.rst]: Could we instead add it to the limitations at the end of the page? I don't think it requires a bigger mention than other limitations.

**Review by pchaigno — CHANGES_REQUESTED** [review]
Thanks for the PR! Not sure how this was missed.

File: Documentation/security/network/encryption-ipsec.rst (hunk 1 above)
Comment (f1ko) [reply Documentation/security/network/encryption-ipsec.rst]: Done 👍

**Review by pchaigno — APPROVED** [review]

**Review by qmonnet — CHANGES_REQUESTED** [review]
Looks good, thanks! Please squash your two commits together.

**Review by qmonnet — APPROVED** [review]
Thanks a lot!

**Comment by qmonnet** [comment]
Probably worth to backport, I'll add the labels

**Pull request merged**